playwright/tests/integration/cms.spec.ts

```diff
@@ -1,6 +1,16 @@
 import { expect, test } from '@playwright/test';
 import { CmsEditor } from '../pageobjects/CmsEditor';
 
+let editor: CmsEditor;
+
+test.beforeEach(async ({ page }) => {
+  editor = await CmsEditor.openNewCms(page);
+});
+
+test.afterEach(() => {
+  editor.deletePmv();
+});
```

Comment:
this will also run if you just open the cms (e.g. from the load data test)
I would create a new test file for this or at least do a describe block and move the beforeEach afterEach side of this describe block.